### PR TITLE
[guava] add more fuzz targets

### PR DESCRIPTION
**Pull request by aschaich**

### CODE REVIEW

File: projects/guava/HashingFuzzer.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+		int k0 = fuzzedDataProvider.consumeInt();
+		int k1 = fuzzedDataProvider.consumeInt();
+
+		HashInputData hashInputData = new HashInputData(fuzzedDataProvider);
+
+		/*
+		 * testHash handles exceptions itself, so this try-block
+		 * only catches exceptions thrown by Hashing's "factory"
+		 * functions, none of which is documented to throw
+		 * exceptions
```
Comment (Alan32Liu) [inline projects/guava/HashingFuzzer.java:162]: Could you please add a period at the end of each comment, thanks!
Same above and below : )

File: projects/guava/UrlEscapersFuzzer.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+public class UrlEscapersFuzzer {
+
+	/*
+	 * these constants are private members copy-pasted from
+	 * com.google.common.net.UrlEscapers
+	 */
+	static final String URL_FORM_PARAMETER_OTHER_SAFE_CHARS = "-_.*";
+	static final String URL_PATH_OTHER_SAFE_CHARS_LACKING_PLUS =
+      "-._~" // Unreserved characters.
```
Comment (Alan32Liu) [inline projects/guava/UrlEscapersFuzzer.java:19]: Unsure if this is proper indentation, could you please check? Thanks.

**Review by Alan32Liu — COMMENTED** [review]
I left some minor comment, could you please take a look? Thanks.

File: projects/guava/UrlEscapersFuzzer.java (hunk 2 above)
Comment (aschaich) [reply projects/guava/UrlEscapersFuzzer.java]: Reintended the file

File: projects/guava/HashingFuzzer.java (hunk 1 above)
Comment (aschaich) [reply projects/guava/HashingFuzzer.java]: done

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**